src/main/java/innovimax/mixthem/operation/ZipOperation.java

```diff
@@ -0,0 +1,23 @@
+package innovimax.mixthem.operation;
+
+import innovimax.mixthem.arguments.ParamValue;
+
+public enum ZipOperation { 
+  DEFAULT_ZIP_SEPARATOR("");
+
+  private ParamValue value;
```

Comment:
why an enum here? There is only one value !

Reply:
We've decided to put constants in enum and group them by topic classes, so we have small enum.
There was only 5 constants in previous class.